src/borg/testsuite/archiver.py

```diff
@@ -4095,6 +4095,42 @@ def test_sort_option(self):
 
         assert all(x in line for x, line in zip(expected, output.splitlines()))
 
+    def test_format_option(self):
+        self.cmd('init', '--encryption=repokey', self.repository_location)
+
+        self.create_regular_file('a_file', size=8)
+        test_archive0 = self.repository_location + '::test0'
+        self.cmd('create', test_archive0, self.input_path)
+
+        os.unlink('input/a_file')
+        self.create_regular_file('a_file', size=33)
+        atime, mtime = 123457520, 234900290
+        os.utime('input/a_file', (atime, mtime))
+
+        test_archive1 = self.repository_location + '::test1'
+        self.cmd('create', test_archive1, self.input_path)
+
+        output_file_mod = self.cmd('diff', test_archive0, "test1")
+        output2 = self.cmd(
+            'diff', '--format',
+            '{newsize:8d} B {oldsize:8d} B {path}{NEWLINE}',
+            test_archive0, "test1")
```

Comment:
as long as you can stay < 120 chars, you can keep it on one line.

Reply:
I made those changes. Only the long lines I kept them splitted since they are > 120 chars plus it's more comfortable in my environment to have not too long lines, if that's not an issue for you